analyticscloud/uploader.py

```diff
@@ -19,7 +20,15 @@ def _stringify(s, encoding, errors):
     if isinstance(s, datetime):
         return s.strftime('%Y-%m-%d %H:%M:%S')
 
-    return unicodecsv._stringify(s, encoding, errors)
+    if s is None:
+        return ''
+    if isinstance(s, unicode):
+        return s.encode(encoding, errors)
+    elif isinstance(s, numbers.Number):
```

Comment:
This is direct inlining of the function from unicodecsv because reasons. I'm willing to fix it but I never even looked at the code I copied to begin with.